src/material-examples/form-field-custom-control/my-tel-input-example.css

```diff
@@ -0,0 +1,21 @@
+div {
```

Comment:
Can these use `css` classes instead of `div` etc.? I know it was like this before, but it's not consistent with the other examples.

Reply:
Sure. Updated.